### PR TITLE
bug fixes for v1beta1 and some typo in README.md

### PR DESCRIPTION
**Pull request by wanyufe**
*metadata and template fix for v1beta1:*

*Description of changes:*
metadata clusterctl needs to be v1alpha3
and templates infrastructure for cloudstack should be v1beta1
some typo in README.md

*Testing performed:*
test-thing passed

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jweite-amazon** [comment]
Would the changes to cluster-template.yaml have to be applied to the other template flavors too?

**Review by maxdrib — APPROVED** [review]

File: templates/cluster-template.yaml [hunk 1]
```diff
@@ -50,13 +50,13 @@ spec:
     preKubeadmCommands:
       - swapoff -a
   infrastructureTemplate:
```
Comment (wongni) [inline templates/cluster-template.yaml:52]: Does 'infrastructureTemplate' work with clusterctl v1.0.0+?
In my case, I had to replace this to the following 
```
  machineTemplate:
    infrastructureRef:
```

**Review by wongni — CHANGES_REQUESTED** [review]
Please update the other template files under template/ folder
- cluster-template-managed-ssh.yaml
- cluster-template-ssh-material.yaml

File: README.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ Assuming your running environment is MacOS:
 
 3. Generate the CAPC cluster spec yaml file
     ```
-    clusterctl generate cluster \
+    clusterctl generate cluster capc-cluster \
```
Comment (jweite-amazon) [inline README.md:139]: Is the cluster name param strictly needed, given that it's defined in the CLUSTER_NAME env var that precedes it?  If it is, then lets remove it from the env vars.  (clusterctl should substitute it into the template either way AFAIK).

**Comment by wanyufe** [comment]
Since the changes/fixes had already been in other PRs, closing this PR.

**Pull request closed without merging**